package secondproject;

public class Exception {

	public static void main(String[] args) {
		try {
		int sum=10/0;
		}
		catch(ArithmeticException e) {System.out.println("not possible");


	}
System.out.println("rest of the code");
}}
